src/io/sampling.jl

```diff
@@ -142,25 +228,18 @@ function sample_guided(d::Domain, n::Int, sampler=SobolSample())::DataFrame
 
     # Remove unguided scenarios as an option
     mod_df = copy(spec_df)
-    guided_col = mod_df.fieldname .== :guided
-    g_upper = mod_df[guided_col, :upper_bound][1]
 
     insertcols!(mod_df, :val, :bounds => copy([d.model[:bounds]...]))
-    mod_df[guided_col, :val] .= 1
-    mod_df[guided_col, :lower_bound] .= 1
-    mod_df[guided_col, :bounds] .= [(1, g_upper)]
-    mod_df[guided_col, :full_bounds] .= [(1, g_upper)]
+    mod_df = adjust_guided_bounds(mod_df, 1)
 
     # Sample without unguided, then revert back to original model spec
     ADRIA.update!(d.model, mod_df)
     samples = _sample(d, n, sampler)
```

Comment:
`_sample()` seems to have been renamed `sample()` at some point, and before alternate distributions were possible.
(meaning this and `sample_cf`, `sample_unguided` are now all broken).

Although we don't seem to do specific cf/unguided/guided sampling any more, might as well take this opportunity and fix it now.

Reply:
so everything called sample() should be called _sample()? I was trying to work out where _sample() was defined and could only find sample() so that's what I used...

Reply:
No, it is now `sample()` - we don't need a separate internal function for our purposes hence why it was renamed

Reply:
right, thanks, I'll change it in sample_cf()